packages/notifications/src/styled/StyledNotification.ts

```diff
@@ -5,36 +5,84 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-import styled from 'styled-components';
 import PropTypes from 'prop-types';
-import classNames from 'classnames';
-import CalloutStyles from '@zendeskgarden/css-callouts';
-import { retrieveComponentStyles, isRtl } from '@zendeskgarden/react-theming';
+import styled, { css, ThemeProps, DefaultTheme } from 'styled-components';
+import { retrieveComponentStyles, getColor } from '@zendeskgarden/react-theming';
 import { VALIDATION_TYPE, ARRAY_VALIDATION_TYPE } from '../utils/types';
+import { StyledTitle } from './content/StyledTitle';
+import { StyledClose } from './content/StyledClose';
+import { StyledBase } from './StyledBase';
 
 const COMPONENT_ID = 'notifications.notification';
 
 export interface IStyledNotificationProps {
   type?: VALIDATION_TYPE;
 }
 
+const closeStyles = (props: ThemeProps<DefaultTheme>) => {
```

Comment:
Feels like these styles should be applied directly on `StyledClose` – leveraging the `hue` prop for all designed combos. If there are only opacity variations, let's get a designer involved to see if there's an easy win for normalizing close button color styling across the board.

Reply:
Could you help paint a picture of how that might look like? Since `Close` is a child provided by the end consumer, the only way to receive a hue in `StyledClose` would be through context. Is that right? Also, in addition to hue, `StyledClose` would need to know which component (`Alert` or `Notification`) it was rendered in so that it can apply appropriate styles. How would `StyledClose` know what component it is a descendant of?